kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-queries/src/main/java/org/kie/server/remote/rest/jbpm/queries/TaskQueryResource.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.remote.rest.jbpm.queries;
+
+import static org.kie.server.api.rest.RestURI.*;
+import static org.kie.server.remote.rest.common.util.RestUtils.*;
+
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.kie.server.api.model.instance.TaskInstanceList;
+import org.kie.server.remote.rest.common.Header;
+import org.kie.server.services.api.KieServerRegistry;
+import org.kie.server.services.jbpm.queries.TaskQueryServiceBase;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("server/" + TASKS_GET_FILTERED_URI)
+public class TaskQueryResource {
+	
+	private static final Logger logger = LoggerFactory.getLogger(TaskQueryResource.class);
+	
+	private final TaskQueryServiceBase taskQueryServiceBase;
+	private KieServerRegistry context;
+	
+	public TaskQueryResource(TaskQueryServiceBase taskQueryServiceBase, KieServerRegistry context) {
+		this.taskQueryServiceBase = taskQueryServiceBase;
+		this.context = context;
+	}
+	
+	
+	//TODO: We now only support standard QueryFilterSpec configurations ... Do we also need to support builders???
+	@POST
```

Comment:
Same comment as above. I could change the method name in "runTaskInstancesQueryWithFilters" if necessary.